docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3.5'
+x-web_env: &web_env
+  environment:
+      CI: "true"
+      SETTINGS__solr__host: solr
+      SETTINGS__solr__collection: 'psul_catalog'
+      SETTINGS__redis__sessions__uri: "redis://redis:6379/3"
+services:
+  redis:
+    image: redis:6.0.6-alpine3.12
+    volumes:
+      - redis-data:/data
+    ports:
+    - "6379:6379"
+  web:
+    tty: true
+    stdin_open: true
+    <<: *web_env
+    depends_on: 
+      - redis
+      - solr
+    command: 
+      - /app/bin/startup
+    build: 
+      context: . 
+      target: dev
+    volumes:
+    - bundle-data:/app/vendor/bundle
+    - node-data:/app/node_modules
+    - type: bind
+      source: ./
+      target: /app/
+    ports:
+      - "3000:3000"
```

Comment:
@whereismyjetpack  
```
ports:
      - "${APP_PORT:-3000}:3000"
```
can we do the same we did for myaccount here?

Reply:
This should be fixed up